src/openms/source/CHEMISTRY/EnzymesDB.cpp

```diff
@@ -307,4 +315,16 @@ namespace OpenMS
       }
     }
   }
+
+  void EnzymesDB::getAllMSGFNames(vector<String>& all_names) const
+  {
+    all_names.clear();
+    for (ConstEnzymeIterator it = const_enzymes_.begin(); it != const_enzymes_.end(); ++it)
```

Comment:
consider using something like: 
for (const Enzyme& e ; const_enzymes_)
{ ... e.getMSGFId() ...) 
for better readability in future PRs